### DISCUSSION TITLE
Enable Multiple Key Keybindings

### DISCUSSION THREAD

**Discussion by jrock2004** [hyprwm/Hyprland#1157] [Q&A] [answered]
So in Xmonad I had a keyboard shortcut set up that would like me hit the following keys to execute a command.

```
SUPER+Control+Shift+4
```

I tried the following:

```
bind = $mainMod, SUPER_Control_L_SHIFT_4, exec, scrot -s ~/foo.png && xclip -selection clipboard -t image/png ~/foo.png && rm ~/foo.png
```

Thanks

**Comment by taylor85345** [comment, marked as answer]
The appropriate syntax here would be:

```
bind = $mainMod_CONTROL_SHIFT,4, exec, scrot -s ~/foo.png && xclip -selection clipboard -t image/png ~/foo.png && rm ~/foo.png
```

Basically all modifier keys go before the first comma, and only the executing key goes after the comma.

Please see the wiki for more information: https://wiki.hyprland.org/Configuring/Binds/#basic
